src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -298,109 +258,15 @@ def handle_active_result(
     result: CheckResult,
     metric_tags: dict[str, str],
 ):
-    organization = detector.project.organization
-
-    if features.has("organizations:uptime-detector-handler", organization):
-        # XXX(epurkhiser): Enabling the uptime-detector-handler will process
-        # check results via the uptime detector handler. However the handler
-        # WILL NOT produce issue occurrences (however it will do nearly
-        # everything else, including logging that it will produce)
-        packet = UptimePacketValue(
-            check_result=result,
-            subscription=uptime_subscription,
-            metric_tags=metric_tags,
-        )
-        process_detectors(
-            DataPacket(source_id=str(uptime_subscription.id), packet=packet), [detector]
-        )
-
-        # Bail if we're doing issue creation via detectors, we don't want to
-        # create issues using the legacy system in this case. If this flag is
-        # not enabled the detector will still run, but will not produce an
-        # issue occurrence.
-        #
-        # Once we've determined that the detector handler is producing issues
-        # the same as the legacy issue creation, we can remove this.
-        if features.has("organizations:uptime-detector-create-issues", organization):
-            return
-
-    uptime_status = uptime_subscription.uptime_status
-    result_status = result["status"]
-
-    redis = _get_cluster()
-    delete_status = (
-        CHECKSTATUS_FAILURE if result_status == CHECKSTATUS_SUCCESS else CHECKSTATUS_SUCCESS
+    packet = UptimePacketValue(
+        check_result=result,
+        subscription=uptime_subscription,
+        metric_tags=metric_tags,
+    )
+    process_detectors(
+        DataPacket(source_id=str(uptime_subscription.id), packet=packet),
+        [detector],
     )
-    # Delete any consecutive results we have for the opposing status, since we received this status
-    redis.delete(build_active_consecutive_status_key(detector, delete_status))
-
-    if uptime_status == UptimeStatus.OK and result_status == CHECKSTATUS_FAILURE:
-        if not has_reached_status_threshold(detector, result_status, metric_tags):
-            return
-
-        issue_creation_flag_enabled = features.has(
-            "organizations:uptime-create-issues",
-            detector.project.organization,
-        )
```

Comment:
This has moved to the detector processor

https://github.com/getsentry/sentry/blob/330b1386def62618f3f0e7562e83d5f50d6b9c17/src/sentry/uptime/grouptype.py#L176-L179